bindings/go/Makefile.am

```diff
@@ -37,6 +37,14 @@ bench: $(GOTMPDIR)
 	$(GO) env -w GO111MODULE=off ; \
 	LD_LIBRARY_PATH=$(UCX_SOPATH):${LD_LIBRARY_PATH} $(GO) test -v -bench=.
 
-all: build
+build-perftest: $(GOTMPDIR)
```

Comment:
1. the Makefile target name should be the resulting binary name, for example "goperftest"
2. all targets that do not create output (e.g run-perftest, all) should be marked as phony:
`.PHONY: all run_perftest test bench`
3. i think the target "build" should also be changed to the name of the resulting binary

Reply:
`build` just builds go bindings but does not result in some lib or binary.

Reply:
doesn't it create any output files in GOOBJDIR?

Reply:
Just temporary files.

Reply:
ok then